extensions/aperture/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "aperture",
+  "title": "Aperture",
+  "description": "Record you screen directly from Raycast",
+  "icon": "extension-icon.png",
+  "author": "jomifepe",
+  "categories": [
+    "Applications",
+    "Media",
+    "Productivity"
+  ],
+  "keywords": [
+    "aperture",
+    "record",
+    "screen",
+    "recorder",
+    "video",
+    "capture",
+    "mp4"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "record.command",
+      "title": "Start/Stop Recording Your Screen",
+      "subtitle": "Aperture",
+      "description": "Start or stop recording your screen using Aperture",
+      "mode": "view",
+      "keywords": [
+        "start",
+        "record",
+        "recording"
+      ]
+    },
+    {
+      "name": "preferences.command",
+      "subtitle": "Aperture Preferences",
+      "description": "Customize your recording preferences",
+      "title": "Preferences",
+      "mode": "view"
+    }
```

Comment:
Wasn't it possible to move the preferences inside the command as actual preferences? As far as it looks, it's possible.

Reply:
I thought about that, the idea of using a command was to have quick access. Personally, I change the options a lot because sometimes I want to record a longer video, and to avoid having a large file I have to lower the quality (fps, scale factor, etc).

Reply:
We usually don't have extensions in the store with a dedicated command to preferences. What if you migrated the commands's preferences to the actual extension's preferences. And then, you could add a secondary action to the list item so that the user can select the options before actually recording the screen (e.g `Start Recording with Options`)

![CleanShot 2023-10-02 at 15 01 29@2x](https://github.com/raycast/extensions/assets/16003285/07a4a5b6-a542-4916-b677-cc30e56396f7)

Note that you can also use the `Configure Command` action to quickly access your preferences:

![CleanShot 2023-10-02 at 14 59 37@2x](https://github.com/raycast/extensions/assets/16003285/013e94c5-a213-49cf-bb8f-5a50ab4e4839)

Reply:
Ok, I'm gonna move them to the preferences 🙂